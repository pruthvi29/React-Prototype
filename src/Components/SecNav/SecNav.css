.SecNav {
  text-align: center;
  font-family: "Segoe UI", Frutiger, "Frutiger Linotype", "Dejavu Sans", "Helvetica Neue", Arial, sans-serif;
  font-weight: bold;
}

.SecNav .pt-fixed {
	width: 20%;
}
.SecNav .pt-button {
	color: #006666 !important;
}
.no-radius {
	border-radius: 0px !important;
}
.SecNav .pt-active {
	background-color: #000000 !important;
	color: #ffffff !important;
}
.SecNav .pt-fill {
	cursor: auto;
	background: linear-gradient(to bottom, #ffffff, rgba(255, 255, 255, 0)) left no-repeat, center no-repeat #f5f8fa !important;
}
.SecNav .SecNavMob h4 {
	background-color: #006666;
	color: #ffffff !important;
	height: 40px;
	margin-bottom: 0px;
    padding-top: 10px;
}
.SecNavMob .pt-icon-menu {
	float: left;
	padding-left: 15px;
}
.SecNav .pt-icon-caret-down {
	top: 19px;
    position: absolute;
    font-size: 30px;
    left: 42%;
    color: #000000 !important;
}
.vertical-menu {
	width: 100%; /* Set a width if you like */
	text-align: left;
	margin-top: 13px;
	display: none;
}

.vertical-menu a {
    background-color: #eee; /* Grey background color */
    color: black; /* Black text color */
    display: block; /* Make the links appear below each other */
    padding: 8px; /* Add some padding */
	text-decoration: none; /* Remove underline from links */
	font-size: 14px;
    font-weight: normal;
    padding-left: 15px;
}

.vertical-menu p {
	background-color: #eee;
	color: black;
	padding: 10px;
	margin: 0px;
	padding-bottom: 0px;
	font-size: 14px;
}

.vertical-menu a:hover {
    background-color: #FFF; /* Dark grey background on mouse-over */
}

.vertical-menu a.active {
    background-color: #4CAF50; /* Add a green color to the "active/current" link */
    color: white;
}
h4:hover *{
    display: block;
}