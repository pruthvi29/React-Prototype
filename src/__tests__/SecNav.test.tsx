import * as React from 'react';
import SecNav from '../Components/SecNav/SecNav';
import '../Components/SecNav/SecNav.css';
const renderer = require('../../node_modules/react-test-renderer');

describe('<AdvocateNames />', () => {
test('Testing Sec Nav Component', () => {
  const component = renderer.create(
    <SecNav></SecNav>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
 });
});