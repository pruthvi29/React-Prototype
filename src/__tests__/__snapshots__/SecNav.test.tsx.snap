// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdvocateNames /> Testing Sec Nav Component 1`] = `
<div
  className="SecNav"
>
  <div
    className="pt-button-group pt-fill SecMenu"
  >
    <button
      className="pt-button pt-align-left no-radius pt-active"
    >
      Dashboard
      <span
        className="pt-icon-standard pt-icon-caret-down"
      />
    </button>
    <button
      className="pt-button pt-align-left"
    >
      Inventory
    </button>
    <button
      className="pt-button pt-align-left"
    >
      Trade History
    </button>
    <button
      className="pt-button pt-align-left"
    >
      Vehicle Estimator
    </button>
    <button
      className="pt-button pt-fixed pt-align-right"
    />
    <button
      className="pt-button pt-fixed pt-align-right no-radius"
    >
      Locator History
    </button>
  </div>
  <div
    className="SecNavMob"
  >
    <h4>
      <span
        className="pt-icon-standard pt-icon-menu hamMenu"
      />
      Dashboard
      <div
        className="vertical-menu"
      >
        <p>
          Main Navigation
        </p>
        <a
          href="#"
        >
          Home
        </a>
        <a
          href="#"
        >
          Applications
        </a>
        <a
          href="#"
        >
          Reports
        </a>
        <a
          href="#"
        >
          Resource Library
        </a>
        <a
          href="#"
        >
          Forms
        </a>
        <a
          href="#"
        >
          News
        </a>
        <a
          href="#"
        >
          Directories
        </a>
        <p>
          Secondary Navigation
        </p>
        <a
          href="#"
        >
          Inventory
        </a>
        <a
          href="#"
        >
          Trade History
        </a>
        <a
          href="#"
        >
          Vehicle Estimator
        </a>
        <a
          href="#"
        >
          Locator History
        </a>
      </div>
    </h4>
  </div>
</div>
`;
